mmv1/products/vertexai/api.yaml

```diff
@@ -103,6 +106,218 @@ objects:
         input: true
         description: |
           Points to a YAML file stored on Google Cloud Storage describing additional information about the Dataset. The schema is defined as an OpenAPI 3.0.2 Schema Object. The schema files that can be used here are found in gs://google-cloud-aiplatform/schema/dataset/metadata/.
+
+# Vertex AI Endpoints
+  - !ruby/object:Api::Resource
+    name: Endpoint
+    base_url: projects/{{project}}/locations/{{location}}/endpoints
+    create_url: projects/{{project}}/locations/{{location}}/endpoints
```

Comment:
Do we not send the endpointId during create? As specified on https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations.endpoints/create